src/components/CardShowCase.js

```diff
@@ -0,0 +1,52 @@
+import {faChevronLeft, faChevronRight} from "@fortawesome/free-solid-svg-icons";
+import React from "react";
+import ProductCardHome from "./ProductCardHome";
+import {FontAwesomeIcon} from "@fortawesome/react-fontawesome";
+import style from "./CardShowCase.module.css";
+import { Link } from "react-router-dom";
+import { useState } from "react";
+
+const CardShowCase = (props) => {
+    let maxItemsPerPage = 4
+    let cardWitdh = 225.88
+    let margin = 5
+    for (let i = maxItemsPerPage; i > 0; i--) {
+        if ((cardWitdh + margin) * maxItemsPerPage >= (window.innerWidth * 0.8)) {
+            maxItemsPerPage --
+        } else break
+    }
+
+const [page, setPage] = useState(0)
+let totalPage = Math.ceil(props.data.length / maxItemsPerPage)
+
+
+ return (
+     <div className={style.cardShowCaseContainer}>
+         <div className={style.cardShowCaseTitle}>
+             <h1>{props.name} <span><Link to={"/all"} className={style.seeMoreLink}>More</Link></span></h1>
+         </div>
+         <div className={style.pagePrevBtnWrapper}>
+           <button onClick={() => {
```

Comment:
- Please make this into the ToggleButton component (already have the JS file in components) as this may be used in other pages